cookbooks/vectortile/templates/default/import-planet.erb

```diff
@@ -7,7 +7,7 @@
 
 set -e
 
-export LUA_PATH='/srv/vector.openstreetmap.org/osm2pgsql-themepark/lua/?.lua;/srv/vector.openstreetmap.org/spirit/?.lua;;'
```

Comment:
This bit is no longer necessary as a change resulting from lua include path changes made in spirit to allow expiry to work